composer.json

```diff
@@ -10,18 +10,40 @@
     }
   ],
   "minimum-stability": "stable",
+  "repositories": [
+    {
+      "type": "vcs",
+      "url": "git@github.com:silinternational/simplesamlphp-module-expirychecker"
```

Comment:
What do these entries in `repositories` do?

Reply:
Pull directly from github instead of packagist. I needed them (or did I?) when I was using the feature branch. Could be removed now, I think, but I was planning to wait until changing to a release number.

Reply:
I think you can simply specify something like `dev-feature/my-feature-name` as the required version, without having to do the `repositories` approach (for what it's worth).

https://getcomposer.org/doc/articles/versions.md#branches